procdiscovery/pkg/inspectors/langdetect.go

```diff
@@ -23,7 +23,8 @@ func (e ErrLanguageDetectionConflict) Error() string {
 }
 
 type inspector interface {
-	Inspect(process *process.Details) (common.ProgramLanguageDetails, bool)
+	Inspect(process *process.Details) (common.ProgrammingLanguage, bool)
+	GetRuntimeVersion(process *process.Details, podIp string) string
```

Comment:
Maybe worth considering returning a  [version object](https://pkg.go.dev/github.com/hashicorp/go-version#Version) instead of a string 
This way we can do checks like GraterThan out of the box